### DISCUSSION TITLE
Is there an option for maximum supported resolution?

### DISCUSSION THREAD

**Discussion by wouldntyouknow** [jellyfin/jellyfin-androidtv#4093] [General]
I have a small pocket projector with a max physical resolution of 480p. The Jellyfin Android TV app works great on it, but it accepts 1080p AV1 content as locally playable for some reason - and of course it's a stuttering mess as there is no HW decoding support for AV1.
On the generic Jellyfin Android app, there is an option for max reported supported resolution where I just can put 720p in for the client and transcoding kicks in automatically - and all is fine. But it is really really inconvenient to use with a remote and no touchscreen - would really prefer the TV variant.
I would had assumed that this option would be present in the Android TV app - but I can't find it anywhere in the options. Is it missing completely or am I just the one who can't find it?
Thanks

**Comment by nielsvanvelzen** [comment]
In the current releases the app detects a few devices (mainly Fire TV/Chromecast) to force a maximum resolution of 1080p.

In the upcoming 0.18 release we ask the system for the maximum allowed resolution of h264 playback and use that as the maximum resolution for all video. This should in theory limit your projector to 480p playback, with the server downscaling everything.

**Comment by wouldntyouknow** [comment]
Wonder if that will work in my case. Thing is, as with many other projectors, they support a higher resolution than their native projecting resolution. I can easily see the projector reporting 1080p h264 capabilities, even the product spec sheet lists this. Why no manual setting?

**Reply by nielsvanvelzen** [reply]
We may add such an option in the future as an advanced setting but we try to auto detect most conditions so playback just works.

**Comment by martiinii** [comment]
I'll pin my comment here: https://github.com/jellyfin/jellyfin-androidtv/issues/2516#issuecomment-2800120541
An addition of this option would be ideal, since there are many android TV that don't report their playback capabilities properly.

**Comment by wouldntyouknow** [comment]
Just peeking in again. I tremendously respect the effort that goes into the project and I also understand the UX related considerations for making it try to set this automatically, but nevertheless, a manual setting would still be really really appreciated :)
